ample/modelling/multimer_definitions.py

```diff
@@ -0,0 +1,87 @@
+DIMER = 'dimer'
```

Comment:
enum, enum, enum ... all of those

```
class NMer(enum.Enum):
    DIMER = 'dimer'
    TRIMER = 'trimer'
    TETRAMER = 'tetramer'
    
    @property
     def MULTIMERS(self):
         return [self.DIMER, self.TETRAMER, self.TRIMER]
```

Reply:
I tried this but I think it overly complicates things and adds lots of unnecessary lines of code. For example we can't just print the MULTIMERS property as the class hasn't been instantiated so I'm going to leave this as is.